vtexdocs/RapiDoc
# Feature/Header Auth Section

#### What is the purpose of this pull request?

To add the correct styles to the authentication page.

Documented [here](https://www.notion.so/vtexhandbook/Estilizar-Header-Auth-RapiDoc-852b4a0e757d4ab19d56bec8ec818f0d)

#### What problem is this solving?

The styles of RapiDoc's authentication page don't match the design of the corresponding section (Header Auth) of the new Developer's Portal, which can be found [here](https://www.figma.com/file/Lx6sXdrw9KEvtSEKWttmv8/Developer-Portal?node-id=3696%3A238901).

#### How should this be manually tested?

Go to [this page](https://deploy-preview-45--elated-hoover-5c29bf.netlify.app/docs/api-reference/catalog#auth) and check against the design at [Figma](https://www.figma.com/file/Lx6sXdrw9KEvtSEKWttmv8/Developer-Portal?node-id=3696%3A238901), keeping in mind only the page title and the API keys' labels and inputs were edited, since the page may cease to exist and turn into a section that persists throughout all pages of the same API Reference.

Also, try typing something in the inputs and going to some other page inside the same API Reference. Then, try going back to the authentication page (you can find it by the name of "Authentication" in the left navbar). You should see the previously typed API keys. In other words, the typed API keys should persist through all pages of the same API Reference. Try reproducing the same steps deleting the previously typed API keys.

#### Screenshots or example usage

#### Types of changes
